addons/csi-nutanix/nutanix-csi-storage.yaml

```diff
@@ -339,15 +339,14 @@ spec:
             - --nodeid=$(NODE_ID)
             - --drivername=csi.nutanix.com
           env:
+            - name: SSL_CERT_FILE
+              value: /etc/kubeone/certs/ca-certificates.crt
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
             - name: NODE_ID
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-{{ if .Config.CABundle }}
-{{ caBundleEnvVar | indent 12 }}
-{{ end }}
```

Comment:
Is this expected?

Reply:
yes. I've replaced those manual out of YAML/kustomize edit with static env/volumes/volumeMounts. volume sourced from configMap however is marked as optional.